app/controllers/search_controller.rb

```diff
@@ -45,6 +55,24 @@ def all_content
 
   def set_search_criteria
     @search_criteria = SearchCriteria.new(params)
+    @additional_search_querys = []
+    if params[:query].present?
+      if params[:query].include? "-"
+        params[:query] = non_hyphenate_query(@search_criteria.query)
+        @additional_search_querys << SearchCriteria.new(params)
+      end
+      params[:query] = results_with_probable_hyphens(@search_criteria.query)
+      @additional_search_querys << SearchCriteria.new(params)
+    end
+  end
+
+  def add_extra_results_for_transformed_queries(type)
+    search_type_object = ExecuteSearch.new.by(type, @search_criteria)
+    @additional_search_querys.each do |added_search_criteria|
```

Comment:
This is cool! Could we pass this into the above functions by returning the value of `additional_search_querys` from this method, instead of using an instance variable? Thanks!

Reply:
I thought we could use it in the view saying "Also showing results for : values in additional search querys" that is why I made it a instance variable. No problem though I ll change it now!

Reply:
Oh! Well, could we pass it into there too by parameter? Let me look in a bit, thanks!